robot_smach_states/src/robot_smach_states/human_interaction/find_person_in_room.py

```diff
@@ -14,21 +14,35 @@
 import robot_smach_states as states
 import robot_smach_states.util.designators as ds
 from robot_skills.util import kdl_conversions
-
+# class CheckIfPersonInRoom(smach.State):
+#     def __init__(self, robot, room):
+#         """
+#
+#         :param robot: robot api object
+#         :param room: room where person should be found
+#         """
+#         smach.State.__init__(self, outcomes=['true', 'false'])
+#         self._robot = robot
+#         self._room = room
+#
+#     def execute(self, userdata=None):
 
 class FindPerson(smach.State):
     """ Smach state to find a person. The robot looks around and tries to recognize all faces in view.
 
         """
     # ToDo: robot only mentions that it has found the person. Doesn't do anything else...
```

Comment:
@jlunenburg You added the ToDo above, what kind of action would you like to see? A retry or a more elaborate action?

Reply:
Fixed in a9a039ba